test/functional/test-custom-element.js

```diff
@@ -1643,11 +1643,28 @@ describes.realWin('CustomElement', {amp: true}, env => {
     it('should be enabled by default', () => {
       stubInA4A(false);
       expect(element.isLoadingEnabled_()).to.be.true;
+      // If loading is enabled, we should be able to toggle the classes
+      element.implementation_.toggleLoading(true);
```

Comment:
Glad the previous patch got reverted. Turns out `element.toggleLoading()` DOES call into `CustomElement::toggleLoading`. You have to call `element.implementation_.toggleLoading()`. Sorry for missing this @jridgewell